parser/parser/cc/grammars/typedruby.ypp

```diff
@@ -3616,6 +3617,10 @@ f_opt_paren_args: f_paren_args
                     {
                       $$ = driver.build.pair_keyword(self, $1, $2);
                     }
+                | tLABEL
+                    {
+                      $$ = driver.build.pair_label(self, $1);
+                    }
```

Comment:
Doesn't this conflict with your other change?

Reply:
Sorry it should've been a simple overwrite but I forgot that this branch was based off of an old master initially. Conflict is resolved now.